### ISSUE TITLE
Override CurationConcerns' PermissionsController to prevent funky layout

### ISSUE THREAD

**Issue by mjgiarlo** [projecthydra/sufia#1731]
### Descriptive summary

When you change permissions and arrive at `/concern/permissions/{ID}/confirm`, you see a view that uses CurationConcerns' styling rather than Sufia's. We should provide a consistent look and feel.

Revert #1726 as part of this work and write a test that fails without the layout override in place. As part of this work, review the [same code in CurationConcerns](https://github.com/projecthydra-labs/curation_concerns/blob/master/app/controllers/curation_concerns/permissions_controller.rb) to see if there are any changes we should make to our local override. (Specifically, do we need to add `authorize! :edit, curation_concern` to the `#copy` method?)

Pertains to Sufia master.

### Expected behavior

Sufia looks like Sufia.

### Actual behavior

![screen shot 2016-03-31 at 2 04 13 pm](https://cloud.githubusercontent.com/assets/131982/14191574/6249c396-f74d-11e5-9348-2f4575782ed6.png)

### Related work

This was removed here: https://github.com/projecthydra/sufia/pull/1726

**Comment by atz** [comment]
Currently seeing:
![screen shot 2016-04-06 at 3 05 21 pm](https://cloud.githubusercontent.com/assets/109954/14334407/1b07b6c6-fc09-11e5-871a-068150bcf831.png)


Good enough to close?

**Comment by mjgiarlo** [comment]
@atz It's still jarring to me. Ultimately we want the CC controller plus Sufia theming. Naively, I thought we should revert #1726 and in so doing review the same code in CurationConcerns to see if there are any changes we should make to our local override. So we'd create a new file called `app/controllers/curation_concerns/permissions_controller.rb` and it'd look like:

```ruby
class CurationConcerns::PermissionsController < ApplicationController
  include CurationConcerns::CurationConcernController

  # This controller is only needed because we override the layout
  # that CurationConcerns sets for it. We should be able to remove this
  # controller once we have updated CC to allow for the layout to be
  # configured.
  layout nil

  def confirm
  end

  def copy
    authorize! :edit, curation_concern
    VisibilityCopyJob.perform_later(curation_concern)
    flash_message = 'Updating file permissions. This may take a few minutes. You may want to refresh your browser or return to this record later to see the updated file permissions.'
    redirect_to [main_app, curation_concern], notice: flash_message
  end

  def curation_concern
    @curation_concern ||= ActiveFedora::Base.find(params[:id])
  end
end
```

**Comment by mjgiarlo** [comment]
Tagging @hectorcorrea and @jcoyne who've worked on this file in the past and who may have ideas about a better way to accomplish overriding the layout of a CC controller. (I see CC has some hooks for themed layouts, but I'm not sure how to hook into that without completely redefining the class. Perhaps CC needs a PermissionsControllerBehavior that we could include and re-theme without needing to copy/paste method definitions?)

**Comment by atz** [comment]
Mixin certainly preferable to straight duplication.

**Comment by mjgiarlo** [comment]
Workaround implemented in #1816. Better approach ticketed in #1815 for the backlog.
